### PR TITLE
Fix copy upstream

### PR DESCRIPTION
**Pull request by eadpearce**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -47,4 +52,4 @@
 
 **Implemented enhancements:**
 
-- Added `feature highlight` and `feature link` fields to `InternationalHomePage` [CMS-1385](https://uktrade.atlassian.net/browse/CMS-1385)
+- Added `feature highlight` and `feature link` fields to `InternationalHomePage` [CMS-1385]
```
Comment (Kenneth-Siu) [inline CHANGELOG.md]: Why remove all the links?

Comment (eadpearce) [reply CHANGELOG.md]: advice from Higher Up. security

File: core/tests/test_upstream_serializers.py [hunk 2]
```diff
@@ -38,7 +42,7 @@ def country_guide_page(article_page):
 
 @pytest.mark.django_db
 def test_upstream_model_serializer(page):
-    data = UpstreamModelSerilaizer.serialize(page)
+    data = UpstreamModelSerializer.serialize(page)
```
Comment (Kenneth-Siu) [inline core/tests/test_upstream_serializers.py]: lol it took quite a few minutes for me to work out what actually changed

Comment (eadpearce) [reply core/tests/test_upstream_serializers.py]: a very sneaky typo

File: core/cache.py [hunk 3]
```diff
@@ -25,6 +25,9 @@
     'components-app': cms.COMPONENTS,
 }
 
+SERVICE_NAMES_TO_ROOT_PATHS = {
+    value: key for key, value in ROOT_PATHS_TO_SERVICE_NAMES.items()}
```
Comment (MichaelWalker) [inline core/cache.py]: could this be simplified to `value: ROOT_PATHS_TO_SERVICE_NAMES.keys()`?

Comment (eadpearce) [reply core/cache.py]: ah i don't think so as i need to reassign both the value and the key so it has to be `.items()`

Comment (MichaelWalker) [reply core/cache.py]: ah sorry - yep misread that line! :)

**Pull request merged**